debian/control

```diff
@@ -0,0 +1,14 @@
+Source: sanoid
+Section: unknown
+Priority: optional
+Maintainer: Dustin L. Howett <dustin@howett.net>
```

Comment:
Whoops! I meant to purge all references to my name, as I'm probably not going to be the maintainer if this merges